examples/test/external_api.mrsh

```diff
@@ -0,0 +1,16 @@
+# func get_revenue_growth(string of stripe API key, int of period in months): float
+
+Connect to stripe.
+
+Define period based on requested period.
+
+Retrieve necessary data from stripe.
+
+Calculate the revenue growth for the requested period.
+
+* get_revenue() = throws an error
+* get_revenue('') = throws an error
+* get_revenue('', 0) = throws an error
+* get_revenue('pk_test_Dt4ZBItXSZT1EzmOd8yCxonL', 3) = 0.03
```

Comment:
Does this test account always have these values?

Reply:
this account does not exist. It is the key stripe used in their examples, and that's the idea of the examples, right? you should be able to use dummy data to express the behaviour.

Reply:
ahh, I see, well I have not tried but also not needed since our tests are mocking those calls in the tests?
![image](https://github.com/alantech/marsha/assets/5898639/e835dec1-1c3e-454a-aee7-4e9c64fbf26a)

Reply:
I've seen it using mocks, but I can assure all generations are using mocks